lib/gauge_field_strength_tensor.cu

```diff
@@ -1,6 +1,7 @@
+#include <typeinfo>
```

Comment:
Do we need typeinfo in every .cu file now? If yes maybe that should be done through a header.

Reply:
This came from one of Kate's recent PRs.

Reply:
Yes, I saw the PR that introduced it in some places. Just wanted to bring it up here. Nothing blocking but would wait for @maddyscientist to comment.

Reply:
This was a fix for GCC-6 that was broken (missed because not in Jenkins).  The need for `typeinfo` comes from using RTTI for identifying the name for `Tunable`.  I can just put this include in `tune_quda.h` and we can remove all the unnecessary `typeinfo` inclusions.

Reply:
fixed